src/titiler/xarray/pyproject.toml

```diff
@@ -38,15 +38,15 @@ dependencies = [
 
 [project.optional-dependencies]
 full = [
-    "zarr>=2,<3",
+    "zarr",
     "h5netcdf",
     "fsspec",
     "s3fs",
     "aiohttp",
     "gcsfs",
 ]
 minimal = [
-    "zarr>=2,<3",
+    "zarr",
```

Comment:
will this work for zarr<3 ? Should that be specified?

Reply:
Yes, I accounted for both the new and old syntax in https://github.com/developmentseed/titiler/blob/e081d0f1452e4a950d8e9c13c939f0c5627956aa/src/titiler/xarray/titiler/xarray/io.py#L81-L89 and tested locally for zarr<3.